docs/pages/installation.mdx

```diff
@@ -485,6 +485,9 @@ $ kubectl exec -i my-pod -- tctl status
 # sending local files via stdin
 $ kubectl exec -i my-pod -- tctl create -f < my-local-file.yaml
 
+# retrieving the teleport service config file from the configmap
+$ kubectl -n <namespace> get configmap teleport-cluster-auth -o jsonpath="{.data['teleport\.yaml']}"
```

Comment:
```suggestion
$ kubectl get configmap teleport-cluster-auth -o jsonpath="{.data['teleport\.yaml']}"
```

None of the examples here specify the namespace, they all assume you're targeting the correct namespace. I would be consistent.